src/nacl/signing.py

```diff
@@ -59,7 +61,20 @@ class VerifyKey(encoding.Encodable, StringFixer, object):
     :param encoder: A class that is able to decode the `key`
     """
 
-    def __init__(self, key, encoder=encoding.RawEncoder):
+    def __init__(self, key, *args, **kwargs):
+        if len(args) == 1:
```

Comment:
I don't like this approach. Why not just do `encoder=None` and if it's not `None` pass that as the encoder after warning that passing encoder is deprecated?

Reply:
I think, if we follow the simpler approach, we wouldn't warn the user
when she calls either
```
VerifyKey(key, <expected_encoder>)
```
or
```
VerifyKey(key, encoder=<expected_encoder>)
```
If you think this is over-protective, I'll change.

Reply:
What is expected encoder in your example? The default arg?

Reply:
Right

Reply:
Even If we
```
    def __init__(self, key, encoder=None):
```
instead of
```
    def __init__(self, key, encoder=encoding.RawEncoder):
```
We would not warn for calls to `VerifyKey(key,  encoder=None)` or `VerifyKey(key, None)`.

Reply:
I'm not worried about people doing that unless `None` did something in the old API already.

Reply:
The good thing is, if we change the default value for `encoder`, the warning will be emitted for all callers predating the change. I'm just unsure about not emitting the warning when the argument is passed-in with the expected value.

Reply:
OK, missed your previous message. `None` was not doing anything in the old API. Changes upcoming.